### PR TITLE
Add `unsafeSVG` directive

### LINKED ISSUE #304: Insert another raw string to svg
I have this code, but it is not work
```js
import { svg } from 'lit-html'
import sealImage from '!!raw-loader!static/assets/signes/seal.svg'

const certificateSealTemplate = svg`<g id="certificateSeal" transform="matrix(0.24731808,0,0,0.24787261,50,30)">
  ${sealImage}
</g>`
```
I can't insert another svg file to svg as string
In the DOM tree I see this

image

How I can fix it?

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #304

### CODE REVIEW

**Comment by justinfagnani** [comment]
@sorvell this is ready for review. The test failures are just HTML expectation differences. I think I fixed them, though it'd be nice to have a more robust matcher. I tried `@open-wc`'s semantic-dom-diff library, but it didn't support SVG.

File: src/directives/unsafe-svg.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import {directive, NodePart, Part, reparentNodes} from '../lit-html.js';
+
+interface PreviousValue {
+  readonly value: unknown;
+  readonly fragment: DocumentFragment;
+}
+
+// For each part, remember the value that was last rendered to the part by the
+// unsafeSVG directive, and the DocumentFragment that was last set as a value.
+// The DocumentFragment is used as a unique key to check if the last value
```
Comment (sorvell) [inline src/directives/unsafe-svg.ts]: It's not clear why the document fragment is used. In what case does is it useful?

File: src/test/directives/unsafe-svg_test.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
```
Comment (sorvell) [inline src/test/directives/unsafe-svg_test.ts]: 2020

File: src/directives/unsafe-svg.ts (hunk 1 above)
Comment (justinfagnani) [reply src/directives/unsafe-svg.ts]: See the 'renders after other values' test. If you render an `unsafeSVG()`, then any other value, then `unsafeSVG()` again with the same parameter, we need to re-render to the part, even though the parameter is the same. Checking that the current value is the right DocumentFragment as part of the dirty-check allows this.

**Comment by justinfagnani** [comment]
Oops, I forgot about #1000. Good news, the implementation is exactly the same. Closing in favor of that.

**Pull request closed without merging**